templates/postgresql_database.yml.erb

```diff
@@ -4,9 +4,14 @@ development: &default
   encoding: utf8
   host: localhost
   min_messages: warning
-  pool: 2
+  pool: <%%= ENV.fetch("DB_POOL", 2) %>
+  reaping_frequency: <%%= ENV.fetch("DB_REAPING_FREQUENCY", 10) %>
   timeout: 5000
 
 test:
   <<: *default
   database: <%= app_name %>_test
+
+production:
+  url:  <%%= ENV.fetch("DATABASE_URL") %>
+  pool: <%%= ENV.fetch("DB_POOL", ENV.fetch("MAX_THREADS", 5) %>
```

Comment:
@schneems That is really interesting. We'll have to give that a try. I created a separate issue to track that: https://github.com/thoughtbot/suspenders/issues/506

For the purpose of this PR, I think it's more incremental to continue to have our separate staging environment. Could end up getting removed soon thereafter but that seems fine.